<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NemocnicaVyber</class>
 <widget class="QWidget" name="NemocnicaVyber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>27</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Výber nemocnice</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>nemocniceCombo</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="nemocniceCombo"/>
   </item>
   <item>
    <widget class="QToolButton" name="pridajButton">
     <property name="toolTip">
      <string>Pridať nemocnicu</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="data.qrc">
       <normaloff>:/icons/list-add.png</normaloff>:/icons/list-add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="zrusButton">
     <property name="toolTip">
      <string>Odstrániť aktuálne vybranú nemocnicu</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="data.qrc">
       <normaloff>:/icons/list-remove.png</normaloff>:/icons/list-remove.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nemocniceCombo</tabstop>
  <tabstop>pridajButton</tabstop>
  <tabstop>zrusButton</tabstop>
 </tabstops>
 <resources>
  <include location="data.qrc"/>
 </resources>
 <connections/>
</ui>
